### PR TITLE
Add KlickSwipe Mod

### PR DESCRIPTION
**Pull request by andrepl**
Add the KlickSwipe Servo-Powered probe dock.

### CODE REVIEW

**Review by jlas1 — COMMENTED** [review]
How does it print, does it require supports or any special instruction?

File: Usermods/KlickSwipe/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+| 6mm x 3mm round neodymium magnets | 1           | Standard Voron size                     |
+
+
+## Assembly
+![Assembly](./images/assembly.gif)
+
+## Configuration
+
+Configure klicky probe as per the usual process, but include the following macros in your config, *after* klicky-macros.cfg:
+
```
[the remark this answers is not in the document]
Comment (andrepl) [reply Usermods/KlickSwipe/README.md:42]: Oops, thanks.  Added.

File: Usermods/KlickSwipe/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+[gcode_macro _DeployDock] # Deploy Arm
+gcode:
+    SET_SERVO SERVO=KlickSwipe ANGLE=90
+    G4 P500
+
+[gcode_macro _RetractDock] # Retract Arm
+gcode:
+    SET_SERVO SERVO=KlickSwipe ANGLE=0
+    SET_SERVO SERVO=KlickSwipe WIDTH=0
+```
```
[the remark this answers is not in the document]
Comment (andrepl) [reply Usermods/KlickSwipe/README.md]: Done!

File: Usermods/KlickSwipe/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# KlickSwipe - A servo-powered retractable dock for Klicky Probe
+
+![KlickSwipe](./images/KlickSwipe%20Render.png)
+
+KlickSwipe is a alternative mount for the klicky probe derived from the [SlideSwipe](https://github.com/chestwood96/SlideSwipe/) probe, but modified specifically for Klicky.  
+It mounts to 1515 extrusion and was designed for use in the V0, but may work in other small printers as well.  
+KlickSwipe is particularly useful when your print head can't reach outside of the bed to attach the probe, as it moves the probe within reach only when necessary.
+
```
[the remark this answers is not in the document]
Comment (andrepl) [reply Usermods/KlickSwipe/README.md:8]: Not particularly.  My name is on the commit, that's credit enough.  It belongs to the community now :)

**Comment by jlas1** [comment]
You forgot to export the STL's, i'll see if i can solve that.

**Comment by andrepl** [comment]
> You forgot to export the STL's, i'll see if i can solve that.

Hmm. I did re-export them. commit 27b074f replaced all of them and renamed one. did I miss something?

**Comment by andrepl** [comment]
I did notice and correct a typo in one of them just now, but also confirmed these are the latest re-exported with your changes.

**Pull request merged**